contrib/!lang/rust/packages.el

```diff
@@ -58,3 +59,12 @@
       :if (configuration-layer/package-usedp 'company)
       :defer t
       :init (push 'company-racer company-backends-rust-mode))))
+
+(defun rust/init-racer ()
+  (use-package racer
+    :if rust-enable-racer
+    :defer t
+    :init (add-hook 'rust-mode-hook
+                    '(lambda ()
+                       (racer-mode)
```

Comment:
If it works for `racer-mode`, I suppose it does for `eldoc-mode` as well… but hooking to the file extension does not seem semantically correct: what we want here is that racer and eldoc are activated with `rust-mode`, but that does not necessarily correspond to only `*.rs` files. For instance, a documentation file with rust code blocks in `mmm-mode`.